lr-cli/cortx_setup/commands/cluster/reset.py

```diff
@@ -0,0 +1,83 @@
+#
+# Copyright (c) 2020 Seagate Technology LLC and/or its Affiliates
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as published
+# by the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU Affero General Public License for more details.
+# You should have received a copy of the GNU Affero General Public License
+# along with this program. If not, see <https://www.gnu.org/licenses/>.
+# For any questions about this software or licensing,
+# please email opensource@seagate.com or cortx-questions@seagate.com.
+#
+
+from cortx_setup.commands.command import Command
+from cortx_setup.commands.common_utils import (
+    get_reset_states
+)
+from provisioner.config import ALL_MINIONS
+from provisioner.commands import (
+    deploy
```

Comment:
I guess we need to call destroy

Reply:
Sorry, just saw its implemented with salt roster file it would be complicated here to do it that way...
The current way looks fine for me.